### DISCUSSION TITLE
[Enhancement] [tree-select] Whether remote search can be added when using lazy load TreeSelect

### DISCUSSION THREAD

**Discussion by liuyifan1** [element-plus/element-plus#11876] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `tree-select`

## Description

Whether remote search can be added when using lazy load TreeSelect，The search data can displayed in select option

### Additional comments

Sorry for my poor English

**Comment by WestFarmer** [comment]
we need this too
